examples/2011_06_07_sampledata_overview.py

```diff
@@ -23,7 +23,7 @@
 # This example shows off our coordinated sample data set.
 # This event got a little bit of press coverage (e.g. `National Geographics
 # <http://news.nationalgeographic.com/news/2011/06/110608-solar-flare-sun-science-space/>`_,
-# `Discover Magazine <http://blogs.discovermagazine.com/badastronomy/2011/06/07/the-sun-lets-loose-a-huge-explosion/#.WVvrBsaZORs>`_)
+# `Discover Magazine <http://blogs.discovermagazine.com/badastronomy/2011/06/07/the-sun-lets-loose-a-huge-explosion/#.WlVnNnWQVxB>`_)
```

Comment:
I believe this one will change with random characters each time we visit it...

Reply:
And maybe we can remove the part ```/#.WlVnNnWQVxB``` without problems, isn't it?

Reply:
maybe... I don't know when the redirect happens.

Reply:
Well without the hash part the link does work. It would be cleaner if nothing else.